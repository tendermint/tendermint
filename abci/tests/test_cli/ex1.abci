echo hello
info
prepare_proposal "abc"
process_proposal "abc"
commit
deliver_tx "abc"
info
commit
query "abc"
deliver_tx "def=xyz"
commit
query "def"
prepare_proposal "preparedef"
process_proposal "replacedef"
process_proposal "preparedef"
prepare_proposal
process_proposal
commit
