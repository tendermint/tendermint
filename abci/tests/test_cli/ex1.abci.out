> echo hello
-> code: OK
-> data: hello
-> data.hex: 0x68656C6C6F

> info 
-> code: OK
-> data: {"size":0}
-> data.hex: 0x7B2273697A65223A307D

> commit 
-> code: OK
-> data.hex: 0x0000000000000000

> deliver_tx "abc"
-> code: OK

> info 
-> code: OK
-> data: {"size":1}
-> data.hex: 0x7B2273697A65223A317D

> commit 
-> code: OK
-> data.hex: 0x0200000000000000

> query "abc"
-> code: OK
-> log: exists
-> height: 0
-> key: abc
-> key.hex: 616263
-> value: abc
-> value.hex: 616263

> deliver_tx "def=xyz"
-> code: OK

> commit 
-> code: OK
-> data.hex: 0x0400000000000000

> query "def"
-> code: OK
-> log: exists
-> height: 0
-> key: def
-> key.hex: 646566
-> value: xyz
-> value.hex: 78797A

